src/EFTools/EntityDesign/TextTemplates/Includes/EF.Utility.CS.ttinclude

```diff
@@ -2285,7 +2285,7 @@ public class DynamicTextTransformation
         Type type = _instance.GetType();
         _write = type.GetMethod("Write", new Type[] { typeof(string) });
         _writeLine = type.GetMethod("WriteLine", new Type[] { typeof(string) });
-        _generationEnvironment = type.GetProperty("GenerationEnvironment", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
Out of curiosity how does this fix the problem?

Reply:
The property is no longer public 😄